client/my-sites/site-settings/wrap-settings-form.jsx

```diff
@@ -68,7 +68,16 @@ const wrapSettingsForm = getFormSettings => SettingsForm => {
 					this.props.isSaveRequestSuccessful &&
 					( this.props.isJetpackSaveRequestSuccessful || ! this.props.jetpackSettingsUISupported )
 				) {
-					this.props.successNotice( this.props.translate( 'Settings saved!' ), {
+					let successNoticeText = this.props.translate( 'Settings saved!' );
+
+					if (
+						this.props.dirtyFields.includes( 'podcasting_category_id' ) &&
+						this.props.fields.podcasting_category_id === '0'
+					) {
+						successNoticeText = this.props.translate( 'Podcasting disabled.' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **2** times:
`translate( 'Disabled podcasting' )` *ES Score: 21*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/podcasting-disable-redirect)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).